features/other_a/preferences_edit.feature

```diff
@@ -250,6 +258,8 @@ Feature: Edit preferences
     And I should not see "Scary tag" within "dl.work.meta"
   When I follow "Show additional tags"
   Then I should see "Scary tag" within "dl.work.meta"
+  When I follow "Scary tag" within "dl.work.meta"
+  Then I should be on the works tagged "Scary tag"
```

Comment:
You can change this to `Then I should be on the "Scary tag" tag page`. What's happening is `Scary tag` isn't a canonical (common) tag, so it doesn't have a works page.  Being taken to the tag page instead is the expected fallback behavior.

Reply:
ah, "common" means the same as canonical? Great! I do need to have a test for ending up on the works page (since the bug is ending up on the detail page) but maybe I can find a different tag to work with!

Reply:
Okay, I think I've resolved this in 84827e9 by marking Scary tag as canonical, but if that's not okay for some reason I'm happy to do something different.